pkg/metrics/metrics.go

```diff
@@ -118,6 +119,14 @@ var (
 		[]string{"endpoint_state"},
 	)
 
+	// ConntrackGC is the number of deleted entries on the conntrack table by
+	// the garbage collector process.
+	ConntrackGC = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
This should be a CounterVec not a GaugeVec

Reply:
Yes, I think it should be a counter. What you describe above is the rate and the rate can be extracted from a counter:
```
rate(conntrack_gc{table="globale"}[5m])
```